src/views/FinalPurchaseConfirmation.vue

```diff
@@ -35,8 +35,8 @@ onMounted(() => {
 <template>
   <component :is="$route.meta.layout || 'div'">
     <template #main>
-      <div className="h-full w-full place-items-center">
-        <div className="text-center font-semibold text-3xl mb-7">
+      <div class="h-full w-full grid grid-rows-5 place-items-center">
```

Comment:
We need to ensure, that this also on the iphone se with a height of 667px and multiple items in basket. If we put here a grid, it can't grow dynamically. So we need to fix the small screen issue, the rest of the code looks good.
![image](https://github.com/augustin-wien/augustin-frontend/assets/404058/c75519d6-faff-4880-a344-a58b9a99015e)